src/lib/components/particles/waffle/index.jsx

```diff
@@ -0,0 +1,37 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+
+const renderUnit = (squares, attributes) => squares ? <rect {...attributes} /> : <circle {...attributes} />
```

Comment:
The JSX might be easier to read if this is a `WaffleUnit` component. It does not need to be exported, but can exist as a 'private' component only available to the Waffle.